Sources/NIOHTTP1/HTTPDecoder.swift

```diff
@@ -115,9 +115,13 @@ public extension ChannelPipeline {
     ///
     /// - parameters:
     ///     - first: Whether to add the HTTP server at the head of the channel pipeline,
-    ///              or at the tail.
+    ///         or at the tail.
+    ///     - supportPipelining: Whether your code can manually handle pipelined requests,
+    ///         or whether it needs requests serialized. If `false` (the default), this
+    ///         call will also insert a `HTTPServerPipelineHandler` to serialize HTTP
+    ///         requests.
     /// - returns: An `EventLoopFuture` that will fire when the pipeline is configured.
-    public func addHTTPServerHandlers(first: Bool = false) -> EventLoopFuture<Void> {
+    public func addHTTPServerHandlers(first: Bool = false, supportPipelining: Bool = false) -> EventLoopFuture<Void> {
```

Comment:
@Lukasa @weissi I guess this change is ok as we have a default param, correct ?

Reply:
@normanmaurer yes 👍